packages/discord-argument/src/lib/structures/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Argument';
```

Comment:
remove this index.ts and just import with an extra path jump elsewhere. That's more future proof to avoid circular references.